spec/models/user_spec.rb

```diff
@@ -69,4 +69,14 @@
       end
     end
   end
+
+  describe "ユーザー削除時の仕様をテストする" do
+    before {
+      @user = user
+      @user.microposts.create!(content: "Lorem ipsum")
+    }
+    it "関連するマイクロソフトも削除されること" do
+      expect { @user.destroy }.to change{ Micropost.count }.by(-1)
```

Comment:
削除もbeforeで行った上で下記の方がテストとしてはやりたいことが明確だと思います。

```suggestion
      expect(Micropost.where(user: user)).to be_empty
```

Reply:
`change` 使った書き方でしか表現できないと思ってました。なるほどこう書けばいいんですね。参考にします。